main/main.cpp

```diff
@@ -1625,6 +1626,9 @@ Error Main::setup(const char *execpath, int argc, char *argv[], bool p_second_ph
 		} else {
 			WorkerThreadPool::get_singleton()->init(worker_threads, low_priority_use_system_threads, low_property_ratio);
 		}
+#else
+		WorkerThreadPool::get_singleton()->init(0, false, 0);
+#endif
```

Comment:
```suggestion
#endif // THREADS_ENABLED
```

Reply:
It would be the only macro to do so in the whole file. :confused: